### ISSUE TITLE
Feedback - Add capture / invisible capture to forms to reduce bot submissions. 

### ISSUE THREAD

**Issue by digitalchild** [WordPress/Learn#2041]
## Type of feedback

#dev#

## Description

On our submission forms that are sent to the Learn Faculty via Help Scout we receive a number of spam and bot submissions. 

https://learn.wordpress.org/report-content-feedback/ 
https://learn.wordpress.org/tutorial-presenter-application/ 
https://learn.wordpress.org/online-workshops/

If we had a capture system in place, it would reduce the work we have going through our queue to find legitimate submissions.

**Comment by jonathanbossenger** [comment]
@adamwoodnz @pkevan do you know if we have an internal .org captcha solution that we can use here?

**Comment by pkevan** [comment]
Isn't akismet running here?

**Comment by jonathanbossenger** [comment]
This was discussed during this week's dev-triage session [here](https://wordpress.slack.com/archives/C01KC5VDQBC/p1701328726162659). 

As far as we can tell, Akismet is indeed in place, but I seem to recall a conversation with @adamwoodnz at some time in the past that sometimes some spam gets through, and it takes Akismet a little while to catch up. So it's a bit of a game of "whack-a-mole".

What I'm not sure of is whether having a Captcha as well is going to help, or be a hindrance to folks submitting the forms....

**Comment by adamwoodnz** [comment]
> I seem to recall a conversation with @adamwoodnz at some time in the past

I don't think it was me sorry. Maybe @tellyworth or @dd32 might be helpful here?

**Comment by pkevan** [comment]
> whether having a Captcha as well is going to help, or be a hindrance

I'm not a huge fan of captchas generally, if we can avoid it that would be better imo.

**Comment by digitalchild** [comment]
There are invisible captures that require no input. The issue I'm seeing is that we are getting form submissions that are still making it through akismet as well as empty submissions.

**Comment by alexstine** [comment]
Google reCAPTCHA is probably the most accessible out there and even it has lots of issues. Simple math captchas likely don't do much against bots today.
